docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -209,30 +205,26 @@ const ModeToggle = () => {
     <Button
       variant="outlined"
       onClick={() => {
-        if (mode === 'light') {
-          setMode('dark');
-        } else {
-          setMode('light');
-        }
+        setMode(mode === 'light' ? 'dark' : 'light');
       }}
     >
       {mode === 'light' ? 'Turn dark' : 'Turn light'}
     </Button>
   );
-};
+}
 ```
 
 Finally, add your newly built `<ModeToggle />` button above `<Sheet />`:
 
 ```diff
```

Comment:
The correct format of the git diff output is to prefix **all the lines with one character**:

- ` `, one space if nothing changed
- `-` if something was removed
- `+` if something was added

It looks strange otherwise.